src/Game/UI/uiPauseMenuDataMgr.h

```diff
@@ -370,6 +370,12 @@ class PauseMenuDataMgr {
     // FIXME: types
     bool useItemFromRecipeAndSave(void* unk, int multiplier, PouchItem* item);
 
+    void grabbedItemStuff(PouchItem* item);
+
+    static void dropItemStuff(const char* name, sead::Heap* heap, int w2_0, bool a4, s64 a5,
```

Comment:
This function seems to be part of some giant utility header (ugh) somewhere in Game/, considering the function itself is located in the middle of hundreds of other random utility functions. Could we move this to e.g. Game/gameItemUtils.h?

Also I think we can figure out what most of the arguments are:

```
void spawnDroppedInventoryItem(
    const char* actor_name,
    sead::Heap* heap,
    int life,
    bool sleep_after_init,
    const uking::act::WeaponModifierInfo* weapon_modifiers,
    bool spawn_via_carry_box,
    float a7, float a8)
```

(a7 and a8 might be rotate_y and rotate_z?)